src/renderers/native/ReactNative/__mocks__/merge.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright 2013-2015, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+'use strict';
+
+// TODO: Replace all callers with spread
+
+var merge = function(a, b) {
+  return {...a, ...b};
+};
+
+// TODO: Replace Object.assign callers with requires
+
+Object.assign = require('Object.assign');
```

Comment:
This is gone now since we have the transform doing this. I guess I didn't push that change.